src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -595,94 +440,152 @@ namespace HVACUnitaryBypassVAV {
-                                                                  NodeType_Air,
-                                                                  NodeConnectionType_Outlet,
-                                                                  1,
-                                                                  ObjectIsParent);
-
-            CBVAV(CBVAVNum).SplitterOutletAirNode = GetOnlySingleNode(
-                SplitterOutletNodeName, ErrorsFound, CurrentModuleObject, Alphas(1), NodeType_Air, NodeConnectionType_Internal, 1, ObjectIsParent);
-
-            CBVAV(CBVAVNum).SplitterOutletAirNode = GetOnlySingleNode(SplitterOutletNodeName,
-                                                                      ErrorsFound,
-                                                                      CurrentModuleObject,
-                                                                      Alphas(1) + "_Splitter",
-                                                                      NodeType_Air,
-                                                                      NodeConnectionType_Inlet,
-                                                                      1,
-                                                                      ObjectIsParent);
+            CBVAV(CBVAVNum).AirOutNode = NodeInputManager::GetOnlySingleNode(Alphas(7),
+                                                                             ErrorsFound,
+                                                                             CurrentModuleObject,
+                                                                             Alphas(1),
+                                                                             DataLoopNode::NodeType_Air,
+                                                                             DataLoopNode::NodeConnectionType_Outlet,
+                                                                             1,
+                                                                             DataLoopNode::ObjectIsParent);
+
+            CBVAV(CBVAVNum).SplitterOutletAirNode = NodeInputManager::GetOnlySingleNode(SplitterOutletNodeName,
+                                                                                        ErrorsFound,
+                                                                                        CurrentModuleObject,
+                                                                                        Alphas(1),
+                                                                                        DataLoopNode::NodeType_Air,
+                                                                                        DataLoopNode::NodeConnectionType_Internal,
+                                                                                        1,
+                                                                                        DataLoopNode::ObjectIsParent);
+
+            if (NumAlphas > 19 && !lAlphaBlanks(20)) {
```

Comment:
New node here to connect to mixer or plenum

Reply:
I guess this could have been just one variable, mixerOrPlenumIndex, or even a bool since it's one or the other or 0. I didn't think of that when I started this ... hindsight. I only use this to trigger sending air to the bypass node, which doesn't care what it's connected to. It's getting late in the game so changing this might slow things down. A follow up would be better and could still go in after the IDD freeze.